code/game/objects/items/devices/radio/headset.dm

```diff
@@ -255,12 +255,17 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	. = ..()
 	AddComponent(/datum/component/wearertargeting/earprotection, list(SLOT_EARS))
 
-/obj/item/radio/headset/ai
+/obj/item/radio/headset/silicon/pai
+	name = "\proper personal Integrated Subspace Transceiver "
+	keyslot = null
+	keyslot2 = null

```

Comment:
are these not the default values?

Also move the base procs above the variants (I don't even see a `/obj/item/radio/headset/silicon` definition?)

Reply:
Oh yeah, I guess those are redundancies. 4dplanner recommended that I set up the /silicon/ base so I went for it rather than trying to go headset/ai/pai.